Added several files.

DarkStarInMe opened the pull request:
Copying from fork

DarkStarInMe committed:
Added several files.

Added several files relating to Seperated True Multiculturalism.

Danarca commented on common/on_actions/imperia_on_monthtly_pulse.txt:
```diff
@@ -0,0 +1,102 @@
+﻿on_imperia_monthly_pulse_country = {
```
`on_imperia_monthly_pulse_country` is triggered from the index on_actions file

Danarca commented on common/institurions/imperia_institutions.txt:
The folder is misspelled

Danarca reviewed (changes requested):
You're also missing some instances of `country_institution_multiculturalism_max_investment_add` from technologies, just add the whole files that contains instances of that term, in case the techs reference other techs in the tech-file... If you get me :p

DarkStarInMe committed:
Added files per feedback.

Fixed a folder being named incorrectly.

Added technologies relating to multiculturalism.

Added on_actions index.
